src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/EventRepository.java

```diff
@@ -10,6 +10,8 @@
 public interface EventRepository extends JpaRepository<Event, Long> {
     List<Event> findByOffenderId(Long offenderId);
 
+    List<Event> findByEventId(Long eventId);
```

Comment:
I think you can just use `findById` - I don't think you need  a new method, just use the one in the superclass. Which also means it will return an `Optional` rather than List which is what you want.

Reply:
Thanks. Done.